cmd/otelcol/main_others.go

```diff
@@ -0,0 +1,28 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build !windows
+
+package main
+
+import "log"
+
+func main() {
```

Comment:
Can we keep this in the main.go and move the helpers in a different file? Main other sounds like a second hand possibility :))

Reply:
Yea having multiple main functions is probably a bad idea. I've restructured this slightly so there's only helper methods in this file - are you okay with how it is now or did you want me to rename the files as well?